re2_commits.jsonl

```diff
@@ -0,0 +1,1109 @@
+{"repository": "re2", "hash": "0a38cba1d9dcfbd713141095582597af700f22e9", "date": "2010-03-02 17:17:51-08:00", "message": "initial release"}
```

Comment:
I believe we have been avoiding adding adding the files created by our programs, if you can delete this from the merge would be appreciated

Reply:
Never mind ignore this, instead can you just output into a folder called uncategorizedData